frb_example/benches/dart/lib/main.web.dart

```diff
@@ -0,0 +1,38 @@
+// ignore_for_file: implementation_imports
+
+import 'dart:convert';
+import 'dart:html';
+
+import 'package:flutter_rust_bridge/flutter_rust_bridge.dart' hide useJSON;
+import 'package:flutter_rust_bridge/src/env/web.dart' show useJSON; // need for const
+
+import 'main.dart' as io;
+import 'package:test_core/src/direct_run.dart';
+import 'package:test_core/src/runner/reporter/expanded.dart';
+import 'package:test_core/src/util/print_sink.dart';
+
+@JS()
+external void close();
+
+void main() async {
+  final result = await directRunTests(
+    // on web, useJSON is not propagated to wasm, so pipe as arg instead
+    () => io.main(useJSON ? ['stub', '--json'] : ['stub']),
+    reporterFactory: (engine) => ExpandedReporter.watch(
+      engine,
+      PrintSink(),
+      color: true,
+      printPlatform: false,
+      printPath: false,
+    ),
+  );
+  _close(result);
+}
+
+void _close(bool result) {
```

Comment:
uber-nit: maybe rename to sendAndClose?

Reply:
it's a copy/pasted file from @Desdaemon repo 😄 ok that can allow to differentiate.

Reply:
done in 5c1638d3